packages/plugin-moxie-token-transfer/src/utils/subgraph.ts

```diff
@@ -142,22 +146,25 @@ export async function getSubjectTokenDetailsBySubjectTokenAddresses(traceId: str
 
     while (retries < maxRetries) {
         try {
+            if (!PROTOCOL_SUBGRAPH_URL) {
```

Comment:
Similar to above, best if we can move this to the top of the function

Reply:
done